src/EditorFeatures/CSharp/AutomaticCompletion/AutomaticLineEnderCommandHandler_Helpers.cs

```diff
@@ -491,7 +517,7 @@ private static bool ShouldAddBraceForLocalFunctionStatement(LocalFunctionStateme
         /// Add brace for ObjectCreationExpression if it doesn't have initializer
         /// </summary>
         private static bool ShouldAddBraceForObjectCreationExpression(ObjectCreationExpressionSyntax objectCreationExpressionNode)
-            => objectCreationExpressionNode.Initializer == null;
+            => objectCreationExpressionNode.Initializer == null && !objectCreationExpressionNode.NewKeyword.IsMissing;
```

Comment:
if the NewKeyword is missing... i'm not sure what state we're in :)

Reply:
so it sounds like a unnecessary check? I am little bit afraid of if the 'new' keyword is missing but if the parser can make sure it exist then this is not needed

Reply:
OK, I removed this.

Reply:
basically, if we don't have a `new` then the parser should have never gotten into even making an object-creeation in the first place.  THat said, to know that, you have to know how the parser is built.  so it's like 50/50 on if the check should stay or not.